inc/osvr/Util/ProcessUtils.h

```diff
@@ -0,0 +1,91 @@
+/** @file
+@brief Header
+
+@date 2016
+
+@author
+Sensics, Inc.
+<http://sensics.com/osvr>
```

Comment:
the indentation is funky - did you use the boilerplate generator or copy and paste? If the latter, I hope you got a new guid at least :D